source/documents/insert.txt

```diff
@@ -55,22 +60,16 @@ field name and value.
 Add New Fields
 --------------
 
-To add a new field for the document, hover over the row number in the
+To add a new field in the document, hover over the row number in the
 dialog (the row number is not part of the document but the dialog
-display) and click on the plus sign add a new field after the field.
-
-.. figure:: /images/compass/insert-document-new-field.svg
-  :figwidth: 621px
+display) and click the plus sign to add a new field after the
+selected row.
 
-You can also add a new field at the end by clicking the Tab from the
-value of the last field.
-
-.. tip::
+.. figure:: /images/compass/insert-document-new-field.png
```

Comment:
Just saw your note in the JIRA ticket. 

> Updated the screenshot to just have the mouse since you need to hover over the field to reveal the plus sign.

True, was just wondering if it's clear in the screenshot. The screenshot is mostly white, as it the little hand icon, so to me it kind of all blends together, but I'll leave it to you to decide.

Reply:
Added a circle around the mouse icon.

Reply:
Excellent.